### PR TITLE
Update quantization.md

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR updates the quantization topic in Semantic Search.

### CODE REVIEW

File: docs/search-views/semantic-search/concepts/quantization.md [hunk 1]
```diff
@@ -21,6 +31,35 @@ One of the most significant advantages of PQ is that it allows for fast distance
...
+| Technique | Description                                             | Advantages                              | Limitations                                     |
+|-----------|---------------------------------------------------------|-----------------------------------------|-------------------------------------------------|
+| Scalar    | Quantizes each component of the vector separately       | Simple, easy to implement               | Might not be the most storage-efficient         |
+| Product (PQ) | Splits the vector and quantizes each sub-vector separately | Better compression, balances accuracy and efficiency | Requires choosing optimal sub-vector sizes      |
+
 ## Which Type to Use
 
 The best quantization type to use depends on your specific use case and the characteristics of your data. For example, if memory usage is a critical concern, then you might opt for a more aggressive quantization method like PQ. If accuracy is more important and memory usage is less of a concern, then you might choose no quantization or a less aggressive method like SQ.
+
+## Examples of Quantization Use Cases
```
Comment (amolth-macrometa) [inline docs/search-views/semantic-search/concepts/quantization.md]: I wonder if we really need to give example here:

1. These are standard techniques, not Macrometa specific so readers can use external documentation. 
2. I am not sure the descriptions are accurate or relevant.

**Review by amolth-macrometa — CHANGES_REQUESTED** [review]

File: docs/search-views/semantic-search/concepts/quantization.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/search-views/semantic-search/concepts/quantization.md]: The idea behind including basic concepts is for people who might not know as much, plus it helps SEO. Do you think we should remove the examples?

Comment (amolth-macrometa) [reply docs/search-views/semantic-search/concepts/quantization.md]: Yes, I am not sure the descriptions are correct

Comment (diana-macrometa) [reply docs/search-views/semantic-search/concepts/quantization.md]: Done

**Review by amolth-macrometa — APPROVED** [review]

**Pull request merged**